packages/app-config-writer/README.md

```diff
@@ -1,6 +1,12 @@
 # @sap-ux/app-config-writer
 
-The `@sap-ux/app-config-writer` adds smartlinks configuration for a SAP UX project. 
+The `@sap-ux/app-config-writer` adds or updates configurations to a SAP UX project.
+
+## Features
+
+### Smart Links configuration
+Smart Link controls can be defined in an application to provide navigation links to other Fiori Applications in a standardized way.
```

Comment:
SAP Fiori applications 
(SAP added and applications lower case)

Reply:
Taken over